app/i18n/locales/de.js

```diff
@@ -277,36 +298,48 @@ export default {
 	saving_profile: 'Profil speichern',
 	saving_settings: 'Einstellungen speichern',
 	Search_Messages: 'Nachrichten suchen',
-	Search: 'Suche',
+	Search: 'Suchen',
```

Comment:
I think here if should be a noun `Suche`, not a verb

Reply:
Thanks for your comment, I changed it